zulip_botserver/setup.py

```diff
@@ -3,12 +3,22 @@
 
 from __future__ import print_function
 
+import os
 import sys
 
+version_py = os.path.join(os.path.dirname(__file__), "zulip_botserver", "__init__.py")
+version_symlink = os.path.join(os.path.dirname(__file__), 'version.py')
+if os.path.islink(version_symlink):
+    os.remove(version_symlink)
+os.symlink(version_py, version_symlink)
+import version
```

Comment:
Why is this created dynamically?  You should be able to just use a static relative path symlink.